index.js

```diff
@@ -87,6 +87,20 @@ new PokemonGoMITM({port: 8081})
 app.use(bp.json());
 app.use(cors());
 
+function renamePokemon(mon) {
+  // don't rename pokemon that have already been renamed
+  if (mon.nickname[2] === '%') return null;
```

Comment:
this won't work for `100%` :).  Also, `100% 15/15/15` is 13 characters long which won't fit in the name field for a pokemon (max 12 characters).

Just something you may want to consider for when you get those perfect mons